var/spack/repos/builtin/packages/py-warpx/package.py

```diff
@@ -73,24 +75,23 @@ class PyWarpx(PythonPackage):
     ]:
         depends_on("warpx@{0}".format(v), when="@{0}".format(v), type=["build", "link"])
 
-    depends_on("python@3.6:3.9", type=("build", "run"), when="@:21.12")
-    depends_on("python@3.6:", type=("build", "run"), when="@22.01:")
+    depends_on("python@3.7:", type=("build", "run"))
```

Comment:
Could remove this line, Spack only supports Python 3.7+ so this will always be true